experiments/yarn/src/main/resources/reference.conf

```diff
@@ -4,7 +4,7 @@ gearpump {
   ###########################
   yarn {
     client {
-      package-path = "/user/gearpump/gearpump.zip"
+      package-path = "/usr/lib/gearpump/gearpump-${version}.zip"
```

Comment:
I mean the file name. {version}.zip is not a valid name. As version will not be replaced by real version